suite-native/module-passphrase/src/components/EmptyWalletInfoSheet.tsx

```diff
@@ -0,0 +1,108 @@
+import { useNavigation } from '@react-navigation/native';
+
+import { Icon, IconName } from '@suite-common/icons';
+import { TxKeyPath } from '@suite-native/intl';
+import {
+    AlertBox,
+    BottomSheet,
+    Box,
+    Button,
+    HStack,
+    Pictogram,
+    Text,
+    VStack,
+} from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import {
+    PassphraseStackParamList,
+    PassphraseStackRoutes,
+    RootStackParamList,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+
+type EmptyWalletInfoSheetProps = {
+    onClose: () => void;
+};
+
+const buttonWrapperStyle = prepareNativeStyle(() => ({
+    width: '100%',
+}));
+
+const listItemStyle = prepareNativeStyle(() => ({
+    width: '90%',
```

Comment:
Why did you decided to define all these dimensions with width percentage? Why not to use padding/margin values?